zerver/tests/test_auth_backends.py

```diff
@@ -817,7 +852,8 @@ def test_log_into_subdomain_when_signature_is_bad(self) -> None:
         data = {'name': 'Full Name',
                 'email': self.example_email("hamlet"),
                 'subdomain': 'zulip',
-                'is_signup': False}
+                'is_signup': False,
+                'next': ''}
```

Comment:
This commit might be more readable if split into two commits, first one that passes through a `next=''` value in the signed data, and then a second that sets it to something that's not `''` and uses it, just because it'd move the boring part of the diff (things like this) into a separate commit.